x-pack/plugins/uptime/public/apps/uptime_page_template.tsx

```diff
@@ -31,20 +32,26 @@ export const UptimePageTemplateComponent: React.FC<Props & EuiPageTemplateProps>
   const {
     services: { observability },
   } = useKibana<ClientPluginsStart>();
+  const { down } = useBreakpoints();
 
+  const isMobile = down('s');
   const PageTemplateComponent = observability.navigation.PageTemplate;
 
+  const mobileCenteredHeader = `
```

Comment:
We put this outside of the component function, so that its identify is stable and doesn't need to be provided as a dependency to useMemo. I forget whether or not hooks evaluate primitives by reference or value.